Makefile

```diff
@@ -964,21 +956,10 @@ all: distrib test_internal
 ifeq ($(UNAME), Darwin)
     MAP_FLAGS= -Wl,-map -Wl,$(BUILD_DIR)/llvm_objects/list.all
     LIST_OUTPUT = > /dev/null
-else
-ifeq ($(OS), Windows_NT)
-    # This is for MinGW: the map file gets written, but the
-    # compilation fails with a file truncation error.  Instead,
-    # we use the old strategy.
-    # Note: we do in fact have to pass in this flag twice.
-    # Note: The grep in this line is necessary in order to avoid file truncation errors
-    # in MinGW.
-    MAP_FLAGS= -Wl,-t -Wl,-t
-    LIST_OUTPUT = 2>&1 | grep "libLLVM" | grep ")" > $(BUILD_DIR)/llvm_objects/list.all
 else
     MAP_FLAGS= -Wl,-Map=$(BUILD_DIR)/llvm_objects/list.all
     LIST_OUTPUT = > /dev/null
 endif
-endif
 
 $(BUILD_DIR)/llvm_objects/list: $(OBJECTS) $(INITIAL_MODULES)
 	# Determine the relevant object files from llvm with a dummy
```

Comment:
This could be more simplified if we don't support MinGW: We can get rid of `LIST_OUTPUT` and in line 985 always pipe to `/dev/null`.

Reply:
Done. Also the removal of xargs later in the file.